templates/kwoc/KWoC is back.txt

```diff
@@ -0,0 +1,20 @@
+KWoC is back!! : It's time for another exciting coding journey
+
+Hello,<br><br>
+
+We hope this email finds you well. We are thrilled to announce that <b>Kharagpur Winter of Code<b> is back this year, and We would like to invite you to be part of this amazing coding extravaganza once again!!<br><br>
+
+Your active involvement in previous editions of KWOC has contributed significantly to its success, and we genuinely appreciate your dedication.<br><br>
+
+<b>Mark your calenders for KWoC {year} :<b><br><br>
+
+<i>Registrations for students as well as projects</i> : {date}
+<i>coding peroid</i>  : {start_date} to {end_date}<br><br>
+
+Stay tuned to our website and social media channels for updates on project selection, mentors, and other important announcements.<br><br>
```

Comment:
I think we can add links to the KWoC Slack and Discord here. And maybe even a link to our fb, insta, twitter, and linkedin.